requirements.txt

```diff
@@ -0,0 +1,48 @@
+BeautifulSoup==3.2.1
```

Comment:
Please delete requirements.txt file. As it doesn't exist in the main repo.

Reply:
ok.